circuits/cpp/barretenberg/ts-headless-acir-test/src/index.ts

```diff
@@ -0,0 +1,113 @@
+import { chromium, firefox, webkit } from "playwright";
+import fs from "fs";
+import { Command } from "commander";
+import { gunzipSync } from "zlib";
+import chalk from "chalk";
+import os from "os";
+
+function formatAndPrintLog(message: string): void {
+  const parts = message.split("%c");
+  if (parts.length === 1) {
+    console.log(parts[0]);
```

Comment:
Can we not use debug instead of console.log in this file?

Reply:
No. Debug is used within the browser context. This captures the output and recreates the output of debug using console.log with color codes. Probably overkill but it's easier to grok the output with color.